### PR TITLE
feat: Restructure APG Support Tables

### PR DESCRIPTION
**Pull request by Paul-Clue**
see issue #https://github.com/w3c/aria-practices/issues/2971

This pull request changes the structure of the support tables. The rows are now AT and browser combinations and the columns are now "MUST" and "SHOULD" assertions. The support percentage now represents the degree of support for "MUST" assertions and "SHOULD" assertions.

### CODE REVIEW

File: server/handlebars/embed/views/main.hbs
Comment (alflennik) [inline server/handlebars/embed/views/main.hbs:46]: Some debugging code here

**Comment by alflennik** [comment]
So I think the design still looks pretty good, but I was wondering @isaacdurazo, given that this is by far the most public representation of the ARIA-AT projects work, if you have any ideas to spruce it up? One possible tweak might be "MUST" and "SHOULD" being all caps, in the past we've gotten accessibility notes about all caps words being hard for people with dyslexia to read. I do wonder if "Must Assertions" or something like that might be easier to understand? But the main thing motivating me to ask for your input is just the `VoiceOver for macOS 14.2 - Safari` part, is seems like we could improve it somehow.

What the new embed looks like embedded in the APG website.

File: server/apps/embed.js
Comment (alflennik) [inline server/apps/embed.js:69]: Is it possible to remove this part of the query?

File: server/apps/embed.js
Comment (alflennik) [inline server/apps/embed.js:261]: There's a lot of dead code still in this file, like this variable is not used anywhere for example. Would it be possible to simplify this file a bit? It's currently getting pretty intimidating. Acknowledging of course that I contributed more than anyone to the current situation.

**Review by alflennik — CHANGES_REQUESTED** [review]
Fantastic work, it's working perfectly. I was even able to test it integrated into a development version of the APG website. I do have a couple notes on the implementation itself. And I pinged Isaac in a comment to weigh in on the design given that this is arguably the most important page in the app.

File: server/handlebars/embed/views/main.hbs
Comment (Paul-Clue) [reply server/handlebars/embed/views/main.hbs]: This is done.

File: server/apps/embed.js
Comment (Paul-Clue) [reply server/apps/embed.js:69]: I don't think so. `testPlanReports` is being used in the code.

File: server/apps/embed.js
Comment (alflennik) [reply server/apps/embed.js:261]: So yes the variable appears 7 times in this file, but all of those uses are only related to setting the variable, not using it. Then if you look at the helpers file and main.hbs you'll see that this variable is passed around a bit but never used. That's what I mean when I say it's dead code.

The larger issue, beyond this one variable, is that a lot of the code in this file is passed around and set but never actually utilized, and I think it's definitely worth the effort to overhaul the whole file and remove all the dead code.

File: server/apps/embed.js
Comment (alflennik) [reply server/apps/embed.js:69]: As I said in another comment, you're right that it's being used. My point is more that it's only being used by dead code and the whole network of dead code present in this file can be removed.

File: server/handlebars/embed/views/main.hbs
Comment (alflennik) [reply server/handlebars/embed/views/main.hbs]: Thx!

**Comment by Paul-Clue** [comment]
This pull request is ready for review.

**Comment by ccanash** [comment]
Latest submitted PR addresses https://github.com/w3c/aria-practices/issues/3017

**Review by alflennik — APPROVED** [review]
Fantastic work!

Please don't merge this yet since we are still awaiting feedback from the APG task force.

**Comment by howard-e** [comment]
The APG TF [discussed this on May 21, 2024](https://www.w3.org/2024/05/21-aria-apg-minutes.html#t03) and confirmed this work is now ready to be merged. I will be merging this later today.

**Comment by mcking65** [comment]
@howard-e 
I thought the tables would update on the APG site when this merged. The old tables are still being rendered. Do we need to change something in the APG content?

**Comment by howard-e** [comment]
@mcking65 this work being merged into the `development` branch makes it immediately available for the staging environment. It will become available to the production environment that APG targets once this is merged to `main` but no content change will be required of the APG then.

**Pull request merged**
